Export 2019 03 05

mTwTm opened the pull request:
l10n for Rocket Pinatubo
See here for the [schedule](https://github.com/mozilla-tw/FirefoxLite/wiki/Release-Schedule) of the release.
Related issues on Rocket:
Adding [6 strings](https://github.com/mozilla-tw/FirefoxLite/commit/7f938bf2ad1ca386188bd4ecaefe3701e0d181df) for [Content Portal Onboarding + Setting](https://github.com/mozilla-tw/FirefoxLite/issues/3173)

Delphine commented on locales/templates/app.pot:
```diff
@@ -134,6 +134,16 @@ msgctxt "about_link_privacy"
+#. The %1$s here is a placeholder for "Life Feed"
+#, c-format
+msgctxt "about_link_life_feed"
+msgid "Learn more about %1$s"
```
Would be good like above to explain what Life Feed is for context

Delphine commented on locales/templates/app.pot:
```diff
@@ -427,6 +442,23 @@ msgid ""
+msgstr ""
+
+msgctxt "lifefeed_learn_more"
+msgid "Learn more"
```
Please provide more context for translation purposes.

mTwTm commented:
Updated, see the following changes:
[The whole pull request](https://github.com/mozilla-l10n/zerda-android-l10n/pull/35/files)
[Changes before review](https://github.com/mozilla-l10n/zerda-android-l10n/pull/35/files/02f76c07bbc4524aa0104f43d43e88fe9722e276)
[Changes after review](https://github.com/mozilla-l10n/zerda-android-l10n/pull/35/files/02f76c07bbc4524aa0104f43d43e88fe9722e276..383aa1992532a4329c3649cf690bf720fabf6595)

Added some comments and remove one duplicate after the review.
